Automattic/wp-calypso
# Site Settings: Introduce SSO card and wrapped security form component

This PR introduces the SSO card in the Security site settings section. It also introduces a wrapped security form component that we will use for adding the new modern cards into the Security settings section. 

Preview of the new SSO card:

![](https://cldup.com/wZPw4LqFmJ.png)

A full list of what this PR suggests is:

* Introducing a new SSO settings card component.
* Introducing a new wrapped security form settings component.
* Embedding the new security form component into the security form section.
* Including the SSO settings in the `filterSettingsByActiveModules` utility and its tests.
* Introducing a new CSS class for security settings wrapper.

To test:

* Checkout this branch
* Go to `/settings/security/$site` where `$site` is a Jetpack site.
* Verify the SSO card looks as on the screenshot.
* Play with the main toggle and the sub settings toggles, verify they work properly when retrieving and storing data, comparing with the values in your wp-admin.
* Try saving with disabled main toggle and verify it still saves correctly.
* Test with dev mode and verify the main toggle and sub toggles are properly disabled.
* Verify all util `filterSettingsByActiveModules` tests still pass:

```
npm run test-client client/state/jetpack/settings/test/utils.js -- --grep="filterSettingsByActiveModules"
```
